app/web/modules/gateway_accounts/createSuccess.njk

```diff
@@ -7,7 +7,7 @@
 	</h1>
 	<div class="govuk-panel__body">
 		Your account reference
-		<br><strong>{{ account.gateway_account_id }}</strong>
+		<br><h5>{{ gatewayAccountIdDerived }}</h5>
```

Comment:
Is there a reason `<h5>` is used here? Not a big thing but this was originally done to reflect the design system https://design-system.service.gov.uk/components/panel/

Reply:
design system as it is, doesn't fit lengthy gateway accounts IDs created for direct debit (eg: `DIRECT_DEBIT:i504oi3cn6im4nc0e0vm2fqa0v`). But I have updated font to `govuk-!-font-size-24` from design system